qemu/tests/cfg/iperf_test.cfg

```diff
@@ -0,0 +1,43 @@
+- iperf_test: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    image_snapshot = yes
+    firewalld_service = disable
+    type = iperf_test
+    fw_stop_cmd = systemctl stop firewalld || service iptables stop
+    linux_compile_cmd = tar zxf %s -C %s > /dev/null ; cd %s ; ./configure > /dev/null; make > /dev/null
```

Comment:
you can use this:
```
    linux_compile_cmd = tar zxf %s -C %s > /dev/null ; cd %s/${iperf_version} ; ./configure > /dev/null; make > /dev/null
```

Then L35 in script `iperf_test.py ` can be removed.

Reply:
`iperf_version` has not been defined before this line, so unable to invoke it.

Reply:
not exactly. It depends on the order of call, not definition,

Reply:
Maybe. If I use `linux_compile_cmd = tar zxf %s -C %s > /dev/null ; cd %s/${iperf_version} ; ./configure > /dev/null; make > /dev/null`, this parameter will be invalid.
```
2019-07-29 04:40:39,822 test             L0329 DEBUG|     iperf_version = iperf-2.0.12
...
...
...
2019-07-29 04:40:39,824 test             L0329 DEBUG|     linux_compile_cmd = tar zxf %s -C %s > /dev/null ; cd %s/${iperf_version} ; ./configure > /dev/null; make > /dev/null
```